diff --git a/node_modules/@react-pdf/renderer/index.d.ts b/node_modules/@react-pdf/renderer/index.d.ts
index 309ad86..1ac6350 100644
--- a/node_modules/@react-pdf/renderer/index.d.ts
+++ b/node_modules/@react-pdf/renderer/index.d.ts
@@ -171,6 +171,7 @@ declare module '@react-pdf/renderer' {
       keywords?: string;
       creator?: string;
       producer?: string;
+      children?: React.ReactNode;
       onRender?: () => any;
     }
 
@@ -200,6 +201,7 @@ declare module '@react-pdf/renderer' {
        * @see https://react-pdf.org/advanced#orphan-&-widow-protection
        */
       minPresenceAhead?: number;
+      children?: React.ReactNode;
     }
 
     interface PageProps extends NodeProps {
